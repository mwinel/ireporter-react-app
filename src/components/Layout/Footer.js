import React, { Component } from 'react';

class Footer extends Component {
  render () {
    return (
      <div className="container">
        <div className="footer">
          <div className="row">
            <p className="copyright-text">Â© 2018, iReporter</p>
          </div>
        </div>
      </div>
    );
  }
}

export default Footer;
