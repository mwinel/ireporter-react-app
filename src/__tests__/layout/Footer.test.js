import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../components/Layout/Footer';

describe('Footer Component', () => {
  let FooterComponent = shallow(<Footer />)
  it('Should match snapshot', () => {
    expect(toJson(FooterComponent)).toMatchSnapshot();
  });
});
