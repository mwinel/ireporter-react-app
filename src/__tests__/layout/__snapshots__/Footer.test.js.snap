// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Should match snapshot 1`] = `
<div
  className="container"
>
  <div
    className="footer"
  >
    <div
      className="row"
    >
      <p
        className="copyright-text"
      >
        Â© 2018, iReporter
      </p>
    </div>
  </div>
</div>
`;
